src/screens/DiscoveryScreenV2/index.js

```diff
@@ -178,24 +251,53 @@ const DiscoveryScreenV2 = ({route}) => {
 
     return <></>;
   };
-
   return (
     <DiscoveryContainer>
       <StatusBar translucent={false} />
-      <Search
-        searchText={searchText}
-        setSearchText={setSearchText}
-        setDiscoveryLoadingData={setIsLoadingDiscovery}
-        isFocus={isFocus}
-        setIsFocus={setIsFocus}
-        setIsFirstTimeOpen={setIsFirstTimeOpen}
-        fetchDiscoveryData={fetchDiscoveryData}
-        onCancelToken={onCancelToken}
-      />
+      {route.name === 'Followings' ? (
+        <Header
+          title={
+            profileState.navbarTitle === 'Search Users'
+              ? "Who you're following"
+              : profileState.navbarTitle
+          }
+          // containerStyle={styles.header}
+          titleStyle={styles.headerTitle}
+          onPress={() => navigation.goBack()}
+          isCenter
+        />
+      ) : (
+        <Search
+          searchText={searchText}
+          setSearchText={setSearchText}
+          setDiscoveryLoadingData={setIsLoadingDiscovery}
+          isFocus={isFocus}
+          setIsFocus={setIsFocus}
+          setIsFirstTimeOpen={setIsFirstTimeOpen}
+          fetchDiscoveryData={fetchDiscoveryData}
+          onCancelToken={onCancelToken}
+          placeholderText={route.name === 'Followings' ? profileState.navbarTitle : undefined}
+        />
+      )}
       <DiscoveryTab
         selectedScreen={selectedScreen}
         onChangeScreen={(index) => setSelectedScreen(index)}
+        tabs={tabs}
       />
+      {route.name === 'Followings' && (
+        <Search
+          searchText={searchText}
+          setSearchText={setSearchText}
+          setDiscoveryLoadingData={setIsLoadingDiscovery}
+          isFocus={isFocus}
+          setIsFocus={setIsFocus}
+          setIsFirstTimeOpen={setIsFirstTimeOpen}
+          fetchDiscoveryData={fetchDiscoveryData}
+          onCancelToken={onCancelToken}
+          placeholderText={route.name === 'Followings' ? profileState.navbarTitle : undefined}
+          hideBackIcon
+        />
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to the addition of a new `Header` component and some modifications to the `Search` component. The code looks fine, but it's hard to say for sure without seeing the full context.



<!-- This is an auto-generated comment by OSS CodeRabbit -->